colorbleed/plugins/maya/publish/validate_vray_translator_settings.py

```diff
@@ -13,6 +13,22 @@ class ValidateVRayTranslatorEnabled(pyblish.api.ContextPlugin):
 
     def process(self, context):
 
+        # Check if there are any vray scene instances
+        # The reason to not use host.lsattr() as used in collect_vray_scene
+        # is because that information is already available in the context
```

Comment:
Should this comment also state why this workaround is implemented? That this code is there because of `pyblish-base#250`?